src/Backoffice/Employees/Domain/Actions/ListEmployeeAction.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Lightit\Backoffice\Employees\Domain\Actions;
+
+use Illuminate\Database\Eloquent\Collection;
+use Illuminate\Database\Eloquent\Model;
+use Lightit\Backoffice\Employees\Domain\Models\Employee;
+use Spatie\QueryBuilder\QueryBuilder;
+
+class ListEmployeeAction
+{
+    /**
+     * @return Collection<int, Model>
```

Comment:
type the Model, here it would be 
```suggestion
     * @return Collection<int, Employee>
```